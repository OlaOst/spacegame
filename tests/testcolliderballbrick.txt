ball.position = [0.0, 0.0]
ball.velocity = [0.0, 1.0]
ball.radius = 0.5
ball.collisionType = Ball

brick.position = [0.0, 0.5]
brick.width = 0.75
brick.height = 0.75
brick.collisionType = Brick
brick.health = 2.0