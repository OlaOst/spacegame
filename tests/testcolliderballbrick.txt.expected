ball.position = [0, -1]
ball.velocity = [0, -1]
ball.radius = 0.5
ball.collisionType = Ball

brick.position = [0, 0.5]
brick.width = 0.75
brick.height = 0.75
brick.collisionType = Brick
brick.health = 1

# TODO: check that we have particle and collision sound spawns